libs/wingc/src/capture.rs

```diff
@@ -84,18 +84,24 @@ impl Visit<'_> for CaptureVisitor {
 	}
 
 	fn visit_function_definition(&mut self, func_def: &FunctionDefinition) {
+		let func_scope = if let FunctionBody::Statements(scope) = &func_def.body {
+			scope
+		} else {
+			return;
+		};
```

Comment:
Rust curious: wouldn't it be a bit nicer to use `let/else` here?

```suggestion
		let FunctionBody::Statements(func_scope) = &func_def.body else {
			return;
		};
```

Reply:
I did not know this is valid syntax, neat.
`cargo fmt` does seem to struggle with it though, but it looks nicer so I'll go through and add it.